app/scripts/events.js

```diff
@@ -891,20 +899,23 @@ define([
                 }
             });
 
+            $workspace.on('submit', '.l-message', function(event) {
+                event.preventDefault();
+            });
+
             $workspace.on('keyup', '.l-message', function() {
-                var val = $('.l-chat:visible .textarea').text().trim();
+                var $textarea = $('.l-chat:visible .textarea'),
+                    $emj = $textarea.find('.j-em'),
+                    val = $textarea.text().trim();
 
-                if (val.length > 0) {
-                    $('.l-chat:visible .textarea').addClass('contenteditable');
+                if (val.length > 0 || $emj.length > 0) {
```

Comment:
$emj.length will be 0 or biggest that 0, so you can 
`(!val.length || !$emj.length)`